packages/js/src/first-time-configuration/first-time-configuration-steps.js

```diff
@@ -292,10 +292,17 @@ export default function FirstTimeConfigurationSteps() {
 			return false;
 		}
 		setSiteRepresentationEmpty( state.companyOrPerson === "emptyChoice" || isCompanyAndEmpty || isPersonAndEmpty );
-		updateSiteRepresentation( state )
+		return updateSiteRepresentation( state )
```

Comment:
Are we sure we need a `return` here? I tried to remove it and it's working fine

Reply:
Yep, you're right, it is superfluous!